UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API/Services/ProductDataService.cs

```diff
@@ -68,21 +88,82 @@ public async Task<ExchangeSetServiceResponse> CreateProductDataByProductIdentifi
             return response;
         }
 
+        private int GetFilesize(SalesCatalogueProductResponse salesCatalogueResponse)
+        {
+            int fileSizeCount = 0;
+            if (salesCatalogueResponse != null && salesCatalogueResponse.ProductCounts.ReturnedProductCount > 0)
+            {
+                foreach (var item in salesCatalogueResponse.Products)
+                {
+                    fileSizeCount += item.FileSize.Value;
+                }
+            }
+            return fileSizeCount;
+        }
+        public bool IsAzureAdB2CSource(AzureAdB2C azureAdB2C)
+        {
+            bool isAzureB2CUser = false;
+            string B2CAuthority = $"{azureAdB2CConfiguration.Value.Instance}{azureAdB2CConfiguration.Value.TenantId}/v2.0/";// for B2C Token
```

Comment:
Ensure camel case for variable declaration. b2CAuthority, adB2CAuthority, audience